tests/testthat/test-repo.R

```diff
@@ -82,6 +82,30 @@ test_that("can find gitlab url", {
   expect_equal(package_repo(desc, list()), ref)
 })
 
+test_that("GitLab subgroups are properly parsed", {
+  target <- "https://gitlab.com/salim_b/r/pkgs/pal/issues/"
+  # 1) from URL field, with and without trailing slash
+  expect_identical(package_repo(desc::desc(text = "URL: https://gitlab.com/salim_b/r/pkgs/pal/"),
+                                list())$url$issue,
+                   target)
```

Comment:
This would be cleaner with a local helper function:

```
issue_url <- function(text) {
  package_repo(desc::desc(text = text), list())$url$issue
}
```

And please use `expect_equal()` instead of `expect_identical()` since there's no floating point involved here.

Reply:
Ok, done.